apps/server/src/modules/oauth/uc/oauth.uc.spec.ts

```diff
@@ -348,4 +350,100 @@ describe('OAuthUc', () => {
 			});
 		});
 	});
+
+	describe('migrateUser', () => {
+		const setupMigration = () => {
+			const code = '43534543jnj543342jn2';
+
+			const query: AuthorizationParams = { code };
+
+			const oauthConfig: OauthConfigDto = new OauthConfigDto({
```

Comment:
Maybe you could use a factory like that: https://github.com/hpi-schul-cloud/schulcloud-server/blob/d51a63b835e9ce85fbf06261e2f3ee9bdc563ba5/apps/server/src/shared/testing/factory/domainobject/school-external-tool.factory.ts#L7-L7

Reply:
we should improve our tests in the next PR that's for OAuthmodule, i think it makes more sense https://github.com/hpi-schul-cloud/schulcloud-server/pull/3891